pkg/plugin/v1alpha1/version.go

```diff
@@ -0,0 +1,29 @@
+package v1alpha1
+
+import (
+	"fmt"
+	"runtime"
+
+	"github.com/spf13/cobra"
+)
+
+// NewCmdVersion returns kmov version command
+func NewCmdVersion() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "version",
+		Short: "Prints version and other details relevant to kubemove plugin",
+		Long: `Prints version and other details relevant to kubemove plugin
+
+Usage:
+kubectl kubemove version
```

Comment:
can we shorten this?
`kubectl km version`
can both kubemove as well as km be supported when used as a plugin?
e.g.
`kubectl km version`
`kubectl kubemove version`

Reply:
yes using the symobolic link